app/src/renderer/components/common/TmSessionImport.vue

```diff
@@ -73,6 +74,9 @@ export default {
     TmFormMsg,
     TmFormStruct
   },
+  computed: {
+    ...mapGetters(["connected"])
```

Comment:
I think we should stick to using only standard features of JavaScript.  We want our code to be used by other developers.

Reply:
?? This is a feature of vuex. Could you elaborate what you mean and what you propose as an alternative?

Reply:
I mean the [object spread property is only a proposal](https://github.com/tc39/proposal-object-rest-spread) and is not part of JavaScript.

As an alternative you can write the following in JavaScript:

```JavaScript
computed: mapGetters(["connected"]),
```

Reply:
We would need to change this in the whole application, as it is used like this in almost every component. I think this change doesn't belong into this PR.
Let's discuss this in the next meeting, ok?

Reply:
I don't understand why it needs to be changed all at once but I'm happy to discuss in the meeting if you like.